packages/autorest.typescript/test/integration/lro.spec.ts

```diff
@@ -256,7 +256,7 @@ describe("LROs", () => {
       } catch (e) {
         assert.equal(
           e.message,
-          "Received unexpected HTTP status code 204 while polling. This may indicate a server issue."
+          "should have thrown instead"
```

Comment:
please update those tests to no longer check for an error. It should check the response object instead.